Tokenization_Samples/README.md

```diff
@@ -1,55 +1,81 @@
-                                                VTS samples
- 
- vts_tokenize
--------------------
+# VTS samples
+
+## tokenize
```

Comment:
Maybe rename this to something like "Tokenization Samples" to avoid the keyword VTS.